solr/config/configsets/all_collection_oag56_v1/managed-schema

```diff
@@ -688,7 +693,8 @@
   <copyField source="relatedStandards.relatedStandardURI" dest="relatedStandards.relatedStandardURI_str" maxChars="256"/>
   <copyField source="relatedStandards.relatedStandardIdentifier" dest="relatedStandards.relatedStandardIdentifier_str" maxChars="256"/>
   <copyField source="delete" dest="delete_str" maxChars="256"/>
-  <copyField source="research_product_metadata_license_urls.name" dest="research_product_metadata_license_urls.name_str" maxChars="256"/>
-  <copyField source="research_product_metadata_license_urls.url" dest="research_product_metadata_license_urls.url_str" maxChars="256"/>
   <copyField source="research_product_metadata_license_urls" dest="research_product_metadata_license_urls_str" maxChars="256"/>
+  <copyField source="bestaccessright" dest="bestaccessright_str" maxChars="256"/>
+  <copyField source="subject" dest="subject_str" maxChars="256"/>
+  <copyField source="journal" dest="journal_str" maxChars="256"/>
```

Comment:
Have you added this field by yourself? Is it a necessary field? It seems not. If that is the case, delete it.

Reply:
It was already in the schema which I copied. I removed it.